vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/test/java/com/vaadin/flow/component/datetimepicker/HasLabelTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2000-2017 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker;
+
+import com.vaadin.flow.component.HasLabel;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HasLabelTest {
```

Comment:
Since `DateTimePicker` uses the locale from the current `UI`, you need to create a new instance of `UI` and set with `UI.setCurrent()`. 

Something like:
```suggestion
public class HasLabelTest {

    @Before
    public void setUp() {
        UI.setCurrent(new UI());
    }

    @After
    public void tearDown() {
        UI.setCurrent(null);
    }
```